docs/5.0/admin-guide.md

```diff
@@ -1268,15 +1268,52 @@ $ tctl status
 ## Ansible Integration
 
 Ansible uses the OpenSSH client by default. This makes it compatible with
-Teleport without any extra work, except configuring OpenSSH client to work with
-Teleport Proxy:
+Teleport without any extra work, except configuring OpenSSH client to work with Teleport Proxy.
 
-* configure your OpenSSH to connect to Teleport proxy and use `ssh-agent` socket
-* enable scp mode in the Ansible config file (default is `/etc/ansible/ansible.cfg` ):
 
-```bash
-scp_if_ssh = True
-```
+There are two recommended scenarios:
```

Comment:
I would recommend organizing this differently. 

There are two methods for OpenSSH clients to **connect** to Teleport Nodes; connecting to a Teleport Node directly over port 3022, or proxying through the Teleport Proxy over port 3023. 

There are two methods for OpenSSH clients to **authenticate** to Teleport Nodes; using Teleport credentials from a `tsh` session, or using a long lived SSH certificate exported from the Teleport Auth server.